cli/image_inspect.go

```diff
@@ -25,9 +25,9 @@ func (i *ImageInspectCommand) Init(c *Cli) {
 		Use:   "inspect [OPTIONS] IMAGE",
 		Short: "Display detailed information on one image",
 		Long:  imageInspectDescription,
-		Args:  cobra.ExactArgs(1),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			return i.runInspect(args)
+		Args:  cobra.MinimumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
I do not think that change `RunE` into `Run` is correct. Here is my reason: we need to know whether this command works or not via `RunE`, since `RunE` would return an error or nil.

With `Run`, we cannot tell whether the command runs into error, this is a confusing situation for users. Users cannot use `echo $?` to judge if it runs well, since the exit code is always 0.

WDYT? @faycheng

Reply:
Yes, you are right.I will rollback to `RunE`.